core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -47,18 +49,22 @@ class WorkflowSubmissionActor(val dataSource: SlickDataSource,
                               val executionServiceCluster: ExecutionServiceCluster,
                               val batchSize: Int,
                               val credential: Credential,
+                              val processInterval: FiniteDuration,
                               val pollInterval: FiniteDuration,
                               val maxActiveWorkflowsTotal: Int,
                               val maxActiveWorkflowsPerUser: Int,
                               val runtimeOptions: Option[JsValue]) extends Actor with WorkflowSubmission with LazyLogging {
 
   import context._
 
-  self ! ScheduleNextWorkflowQuery
+  self ! ScheduleNextWorkflow
 
   override def receive = {
-    case ScheduleNextWorkflowQuery =>
-      scheduleNextWorkflowQuery
+    case ScheduleNextWorkflow =>
+      scheduleNextWorkflowQuery(pollInterval + (scala.util.Random.nextInt(1000) milliseconds))
```

Comment:
moving it is a good idea, but I disagree that it should be a function of the timespan. For long intervals it would be too long and for short, too short.